src/components/DateRangePicker.jsx

```diff
@@ -265,6 +282,34 @@ export default class DateRangePicker extends React.Component {
     }
   }
 
+  /* istanbul ignore next */
+  responsivizePickerPosition() {
+    const { anchorDirection } = this.props;
+    const { dayPickerContainerStyles } = this.state;
+
+    const containerClientRect = this.dayPickerContainer.getBoundingClientRect();
+    const documentBodyStyles = window.getComputedStyle(document.body);
```

Comment:
This call is fairly expensive because it traverses the entire cascade and builds CSSDeclaration. Maybe we can call `responsivizePickerPosition` inside `requestAnimationFrame`, what do you think?